src/System.Security.Cryptography.X509Certificates/tests/PublicKeyTests.cs

```diff
@@ -151,6 +166,35 @@ public static void TestKey_RSA384_ValidatesSignature()
             }
         }
 
+        [Theory, MemberData(nameof(BrainpoolCurves))]
+        public static void TestKey_ECDsabrainpool_PublicKey(byte[] curveData, byte[] notUsed)
```

Comment:
Ah; okay.  I think when I got confused by this I decided that there wasn't anything using it at all; but it's used by the next method.

Perhaps a comment up in `BrainpoolCurves` saying that the second line is a signature block for ASCII("Hello") would be useful.

// Produces a tuple of a Brainpool Curve-based certificate and the signature output for the ASCII-encoded string "Hello".

or something of that nature.